release-notes/opensearch-sql.release-notes-1.2.1.0.md

```diff
@@ -0,0 +1,6 @@
+### 2021-12-10 Version 1.2.1.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.1
+
+### Bug fixes
+
+* Bumping log4j to 2.11.2 ([#324](https://github.com/opensearch-project/sql/pull/324))
```

Comment:
should we change the version number here as well?  2.11.2->2.15.0

Reply:
good point.. updated